core/services/keystore/keys/starkkey/export.go

```diff
@@ -3,35 +3,36 @@ package starkkey
 import (
 	"github.com/ethereum/go-ethereum/accounts/keystore"
 
+	stark "github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/keys"
 	"github.com/smartcontractkit/chainlink/core/services/keystore/keys"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
 const keyTypeIdentifier = "StarkNet"
 
 // FromEncryptedJSON gets key from json and password
-func FromEncryptedJSON(keyJSON []byte, password string) (Key, error) {
+func FromEncryptedJSON(keyJSON []byte, password string) (stark.StarkKey, error) {
 	return keys.FromEncryptedJSON(
 		keyTypeIdentifier,
 		keyJSON,
 		password,
 		adulteratedPassword,
-		func(_ keys.EncryptedKeyExport, rawPrivKey []byte) (Key, error) {
-			return Raw(rawPrivKey).Key(), nil
+		func(_ keys.EncryptedKeyExport, rawPrivKey []byte) (stark.StarkKey, error) {
+			return stark.StarkRaw(rawPrivKey).Key(), nil
```

Comment:
Ok for now to get it working but these `stark.StarkKey` & `stark.StarkRaw` types we can structure/name better.

Reply:
fixed these up back to using `stark.Key` `stark.Raw` and `stark.OCR2Key`